source/manual/rebooting-machines.html.md

```diff
@@ -103,6 +105,24 @@ There is a Fabric task to schedule a machine for downtime in Nagios for
 > manual entry on [adding disks](/manual/adding-disks-in-vcloud.html) for
 > more info.
 
+## Rebooting Cache machines in AWS
+
+The `cache` machines run the `router` app which handles live user traffic.
+To safely reboot them without serving too many errors to users, we must
+remove them from the AWS load balancer target group before rebooting:
+
+1. Login to AWS Console. Switch to the prod environment (`gds aws govuk-production-<your-role> -l`)
```

Comment:
I'd probably say switch to the "right" or "relevant" environment, as you'll need to be in Staging for rebooting machines in Staging.